tests/containers/podman_integration.pm

```diff
@@ -70,6 +70,7 @@ sub run {
     install_packages(@pkgs);
     install_htpasswd if is_sle_micro;
     install_ncat;
+    install_pasta unless is_tumbleweed;
```

Comment:
Why is this excluded?

Reply:
Only available on Tumbleweed for now.

Reply:
And MicroOS?

Reply:
We're not yet testing it on MicroOS but let me add it.

Reply:
```suggestion
    install_pasta unless is_tumbleweed; # requires podman > 5.0
```